gusto/diagnostics.py

```diff
@@ -326,145 +322,209 @@ def compute(self, state):
         return self.field.interpolate(Pi)
 
 
-class Theta_e(DiagnosticField):
-    name = "Theta_e"
+class Sum(DiagnosticField):
+
+    def __init__(self, field1, field2):
+        super(Sum, self).__init__(required_fields=(field1, field2))
+        self.field1 = field1
+        self.field2 = field2
+
+    @property
+    def name(self):
+        return self.field1+"_plus_"+self.field2
 
     def setup(self, state):
         if not self._initialised:
-            space = state.spaces("CG1", state.mesh, "CG", 1)
-            super(Theta_e, self).setup(state, space=space)
+            space = state.fields(self.field1).function_space()
+            super(Sum, self).setup(state, space=space)
```

Comment:
you can just use `super().setup...`

Reply:
Oh yes you're right. I didn't actually change this bit of the code so I'm not sure why git is showing it as a change!